ppr-ui/src/views/newMhrRegistration/MhrRegistration.vue

```diff
@@ -18,7 +18,7 @@
     <BaseDialog
       :setOptions="outOfDateOwnersDialogOptions(getMhrInformation.mhrNumber)"
       :setDisplay="showOutOfDateDraftDialog"
-      @proceed="handleOutOfDateDialogResp"
+      @proceed="handleOutOfDateDialogResp($event)"
```

Comment:
The event is already inferred. I'm not sure this changes anything

Reply:
Ok, let me try again without this change.

Reply:
Yes, it works, thanks for reminder, as I was wondering why no event. Closing this PR.

Reply:
You bet, no prob at all. A nuance of vue templating!